apps/web/src/state/farmsV3/hooks.ts

```diff
@@ -178,22 +179,23 @@ export const useFarmsV3 = ({ mockApr = false }: UseFarmsOptions = {}) => {
 
 export const useStakedPositionsByUser = (stakedTokenIds: bigint[]) => {
   const { address: account } = useAccount()
+  const { chainId } = useActiveChainId()
   const masterchefV3 = useMasterchefV3()
 
   const harvestCalls = useMemo(() => {
-    if (!account) return []
+    if (!account || !SUPPORT_FARMS.includes(chainId)) return []
```

Comment:
I think we have another constant variable in farm package?

Reply:
done